### DISCUSSION TITLE
Retiarii support for tensorflow

### DISCUSSION THREAD

**Discussion by yao-peng-lyb** [microsoft/nni#4605] [Q&A] [answered]
Hi I am trying to figure out if the current version of Retiarii supports tensorflow. From roadmap thread I read here:
https://github.com/microsoft/nni/discussions/3744
It seems the support is available. However on this page:
https://nni.readthedocs.io/en/latest/NAS/QuickStart.html#define-your-model-space
it also says,
`Currently, PyTorch is the only supported framework by Retiarii, and we have only tested PyTorch 1.7 to 1.10.`

Some clarification will be really appreciated. Thank you.

**Comment by ultmaster** [comment, marked as answer]
It's planned, but currently we are limited on hands.

Therefore, I'm afraid the answer is no. Retiarii is NOT supported for tensorflow now.
